### DISCUSSION TITLE
Which bootscore elements are marked up by which funtionally bootstrap colors?

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#751] [Q&A]
As demonstrated in https://bootscore.me/documentation/theme/css-scss-compiler/, bootScore uses the 'functional colors' defined by Bootstrap: https://getbootstrap.com/docs/5.2/customize/color/. These are 'functional colors' in meaning, they are intended to be used in specific contexts: The 'danger'-color shall signal: stop, the 'warn'-color shall say: 'caution', and the 'success'-color shall say 'everything ok'. 

So, it is intended, that a theme uses these colors in the respective contexts. bootscore is a theme. So, is there anywhere a list of site elements that bootscore sees as functionally working elements and marks up with the respective colors? For example: Changing the primary colors changes the color of the headlines and the links (but not the color of the text and so on). So, for which elements are used the secondary color etc. etc.?

Many thanks for your anyswers

**Comment by crftwrk** [comment]
Theme comes with default Bootstrap with no modifications in components or colors https://bootscore.me/documentation/bootstrap-components/. Customizing Bootstrap is the developers job.

There is no list where which color is used because [bootscore.me](https://bootscore.me) is a self-explaining demo page where you can click through and see/inspect which color in which part is used. Go to the shop area, open empty cart, add/remove products, add them twice, go to checkout and fill out some fields and leave some empty. You will get a feedback with info, danger or success colors and can see how they work.
